data_structures/binary_tree/binary_search_tree.py

```diff
@@ -2,15 +2,20 @@
 A binary search Tree
 """
 
+from __future__ import annotations
+
```

Comment:
```suggestion
```
This should not be needed as we use latest Python for testing and writing code which as of now is `3.10.*`

Reply:
Actually, this might still be required, please verify.

Reply:
Yaa, this is not needed.